cliquet/__init__.py

```diff
@@ -130,6 +131,9 @@ def includeme(config):
     Service.cors_origins = tuple(aslist(cors_origins))
     Service.default_cors_headers = ('Backoff', 'Retry-After', 'Alert',
                                     'Content-Length')
+    cors_max_age = settings['cliquet.cors_max_age_seconds']
+    Service.cors_max_age = int(cors_max_age) if cors_max_age else None
```

Comment:
nit: I prefer to have the "if" at the beginning of the line to ease readability.